react/Spinner/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { translate } from '../helpers/I18n'
+import classNames from 'classnames'
+
+import styles from './styles'
+
+// === Component Options ===
+// loadingType = message type from locale [string] (default: '')
+// middle      = Component is centered both vertically & horizontally [bool] (default: false)
+// noMargin    = Component has no margin [bool] (default: false)
+// color       = Color of the spinner (available colors: blue, grey, white, red) [string] (default: 'blue')
+// size        = Size of the spinner (available sizes: tiny, small, medium, large, xlarge, xxlarge) [string] (default: 'medium')
+
```

Comment:
I am not a big fan of comments in code. Let's imagine another developer operates on the code and change something, but does not update the corresponding comments. Then, later on, another developer operates on the code and see something strange:
> the comment says you set a title with `title` but it does not work. And when he looks closer in the code, he sees that to set a title you need to use a `name` attribute. What's going on, there? What a mess!

That is why React offers a propTypes attributes to let developers or typechecker knows what the code is waiting for. See https://facebook.github.io/react/docs/typechecking-with-proptypes.html#proptypes for more information.